mu-plugins/blocks/favorite-button/render.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Output the favorite button block.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\MU_Plugins\Favorite_Button_Block;
+
+$settings = get_block_settings( $block->context['postId'] );
+if ( ! $settings ) {
+	return '';
+}
+
+$user_id = get_current_user_id();
+$show_count = $attributes['showCount'] ?? false;
+$variant = $attributes['variant'] ?? 'default';
+
+if ( ! $user_id && ! $show_count && 'small' !== $variant ) {
```

Comment:
Do we display anything if the user is logged out? I think I get the state this is aimed at, but I wonder if we should bail early if there is no user_id

Reply:
Simplified condition slightly by removing the variant check: https://github.com/WordPress/wporg-mu-plugins/pull/635/commits/2dcb8eecb6a3f76c645a82097251d29dc970caf1